web/cobrands/fixmystreet/base.scss

```diff
@@ -33,69 +33,102 @@
 }
 
 .confirmation-header {
-  padding: 2em 1em;
+  padding: 2em 0;
+  padding-top: 140px;
+  text-align: center;
+  background: transparent url(/cobrands/fixmystreet/images/tick-in-circle-100px.png) center 1em no-repeat;
```

Comment:
If you want this (or any of this CSS) to be available to all cobrands (unlike the fixmystreet.com specific next steps) as you mention in the PR text, then it needs to be in the web/cobrands/sass/base or layout SCSS files, not the fixmystreet.com only ones here).